mqtt.cpp

```diff
@@ -387,7 +364,7 @@ int OSMqtt::_init(void) {
 
 	if (mqtt_client) { mosquitto_destroy(mqtt_client); mqtt_client = NULL; };
 
-	mqtt_client = mosquitto_new("OS", true, NULL);
+	mqtt_client = mosquitto_new(NULL, true, NULL);
```

Comment:
Note that I changed this to `NULL` as mosquitto will generate a random Id thus allowing multiple OS to connect to the same mqtt server. If PR #131 is merged then we have the option of using `load_mac_address()` in the same we do for Arduino.